inc/querysubquery.class.php

```diff
@@ -37,19 +37,22 @@
 /**
  *  Sub query class
 **/
-class QuerySubQuery {
+class QuerySubQuery extends AbstractQuery {
    private $dbiterator;
    private $allowed_operators = ['IN', 'NOT IN'];
-   private $operator = 'IN';
+   private $operator;
```

Comment:
OK, but if you do this change on default operator, you have to define the `IN` operator in constructor call here:
 - https://github.com/glpi-project/glpi/blob/efa005f7e95c75fe7cff4c15264c433fb5e26151/inc/group.class.php#L618
 - https://github.com/glpi-project/glpi/blob/efa005f7e95c75fe7cff4c15264c433fb5e26151/inc/group.class.php#L660
 - https://github.com/glpi-project/glpi/blob/efa005f7e95c75fe7cff4c15264c433fb5e26151/inc/plugin.class.php#L780
 - https://github.com/glpi-project/glpi/blob/efa005f7e95c75fe7cff4c15264c433fb5e26151/inc/plugin.class.php#L801
 - https://github.com/glpi-project/glpi/blob/efa005f7e95c75fe7cff4c15264c433fb5e26151/inc/plugin.class.php#L821

Reply:
The goal is to have nothing as default operator